WebCryptoAPI/derive_bits_keys/hkdf.js

```diff
@@ -232,13 +232,6 @@ function run_test() {
 
             });
         });
-
-        done();
-    }, function(err) {
-        subsetTest(test, function(test) {
-            assert_unreached("setUpBaseKeys failed with error '" + err.message + "'");
-        }, "setUpBaseKeys");
-        done();
```

Comment:
I assume that the function promise_test will catch this failure, I am unsure if the error message should be retained or not.  What will be reported in the event the function setUpBaseKeys fails the promise?

Reply:
The harness will report the promise rejection message directly